HW0/P4/P4.py

```diff
@@ -29,49 +29,118 @@
     #
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
+
```

Comment:
Because this is all within the if statement on line 6 you'll want to keep at least one tab indent throughout, more if you have more conditionals/loops